assets/home/sections/quick-links.js

```diff
@@ -2,21 +2,135 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, external } from '@wordpress/icons';
+import { useState, useRef } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
+import { Grid, Col } from '../grid';
+
+const quickLinksSpecialMapping = {
+	'sensei://install-demo-course': ( { setTitle, originalTitle, timer } ) => {
+		setTitle( 'Installing' );
+		const installedMessage = 'Installed!';
+		const clear = () => {
+			if ( timer.current ) {
+				clearTimeout( timer.current );
+			}
+			timer.current = 0;
+		};
+		const run = () => {
+			setTitle( ( oldTitle ) => {
+				if ( installedMessage === oldTitle ) {
+					clear();
+					return originalTitle;
+				}
+				if ( ! oldTitle.includes( '...' ) ) {
+					timer.current = setTimeout( run, 500 );
+					return oldTitle + '.';
+				}
+				timer.current = setTimeout( run, 2000 );
+				return installedMessage;
+			} );
+		};
+		clear();
+		timer.current = setTimeout( run, 500 );
+	},
+};
 
 /**
- * Quick Links section component.
+ * Component that shows a Quick Link.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.item The item to show.
+ */
+const QuickLink = ( { item } ) => {
+	let { url, title: originalTitle } = item;
+	const [ title, setTitle ] = useState( originalTitle );
+	const timer = useRef( 0 );
+
+	let onClick = null;
+
+	const callback = quickLinksSpecialMapping[ url ];
+	if ( !! callback ) {
+		url = '#';
+		onClick = ( e ) => {
+			e.preventDefault();
+			callback( { setTitle, originalTitle, timer } );
+		};
+	}
+	const isExternal = url && url !== '#' && ! url.includes( '/wp-admin/' );
+	return (
+		<li>
+			{ /* eslint-disable-next-line react/jsx-no-target-blank */ }
+			<a
+				href={ url }
+				target={ isExternal ? '_blank' : null }
+				onClick={ onClick }
+			>
+				{ title }
+				{ isExternal ? (
+					<Icon icon={ external } size={ 16 } fill="currentColor" />
+				) : null }
+			</a>
+		</li>
+	);
+};
+
+/**
+ * A column on the Quick Links section.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data The data related to the column of quick links.
  */
-const QuickLinks = () => (
-	<Section title={ __( 'Quick Links', 'sensei-lms' ) }>
-		<ul>
-			<li>Test</li>
-		</ul>
-	</Section>
+const QuickLinksColumn = ( { data } ) => (
+	<ul>
+		<li>
+			<b> { data.title }</b>
```

Comment:
I think it would make more sense to use an `h3` here instead of a `b`. WDYT?

Reply:
Also, probably it could be out of the `ul`, since it's not one of the links.

Reply:
Although I agree with you considering the semantics of the tags, I feel some mixed feelings about this.

Right now, the layout of the section is like this:

![Current status](https://user-images.githubusercontent.com/529864/195954474-b1338aa3-9f3f-4e40-8078-692b49eb65d4.png)

If I just move the `b` outside the `ul`, it shows like this:

![With the title outside the list](https://user-images.githubusercontent.com/529864/195954537-2c33c4a4-8742-486c-850b-53d82f4e4374.png)

Notice the bigger margin between the title and the list.

Now, if I transform the `b` into a `h3`, it shows like this:

![With h3 and outside the list](https://user-images.githubusercontent.com/529864/195954620-947dbc4f-e4e7-4091-8803-52acef082e8d.png)

Notice that the title is waaay bigger and the margin looks...even bigger?

So, WDYT? Should I still change this? :thinking: 

Also, I just noticed that I forgot to remove the underline of the links to match the design. Awesome. I'll change that.

Reply:
I adjusted that in [07414f5](https://github.com/Automattic/sensei/pull/5896/commits/07414f5ab50230f9a24113ec83888e6fae1e01fe) - Can you take a look? :)

Reply:
Awesome! Thank you! =)